data/json/recipes/other/parts.json

```diff
@@ -402,24 +402,58 @@
   },
   {
     "type": "recipe",
-    "activity_level": "MODERATE_EXERCISE",
-    "result": "cable",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "copper_rod",
     "category": "CC_ELECTRONIC",
     "subcategory": "CSC_ELECTRONIC_COMPONENTS",
     "skill_used": "fabrication",
-    "skills_required": [ "electronics", 2 ],
-    "difficulty": 5,
-    "time": "3 m",
+    "difficulty": 4,
+    "time": "5 m",
     "autolearn": true,
-    "charges": 1,
+    "charges": 5,
     "qualities": [
       { "id": "ANVIL", "level": 1 },
       { "id": "HAMMER", "level": 2 },
-      { "id": "CHISEL", "level": 3 },
-      { "id": "CUT", "level": 1 }
+      { "id": "CHISEL", "level": 3 }
+    ],
+    "tools": [ [ [ "tongs", -1 ] ], [ [ "swage", -1 ] ], [ [ "surface_heat", 2, "LIST" ], [ "forge", 2 ], [ "oxy_torch", 2 ] ] ],
+    "components": [ [ [ "copper", 2 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "result": "copper_wire",
+    "category": "CC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_COMPONENTS",
+    "skill_used": "fabrication",
+    "difficulty": 3,
+    "//": "This assumes an automatic wire belt.",
+    "time": "40 m",
+    "autolearn": true,
+    "//2": "Dividing the diameter by about 3.46 extends it by the square, ~12 times.",
+    "charges": 600,
```

Comment:
I think it will produce ~900 charges assuming conservation of mass?

Reply:
5 `copper_rod`s is 500ml of copper, which translates to 50 copper wire. Multiplying by a factor of 12 gets to 600 charges. Of course, this doesn’t take into account [Poisson’s ratio](https://en.m.wikipedia.org/wiki/Poisson%27s_ratio).

Reply:
I'm not entirely sure of your math here, but 600 charges of copper wire is only 330ml or 3kg, which is far less than the 500ml or 4465g of 5 copper rods.